librosa/filters.py

```diff
@@ -449,7 +449,7 @@ def constant_q(sr, fmin=None, n_bins=84, bins_per_octave=12, tuning=0.0,
     for i in np.arange(n_bins, dtype=float):
 
         freq = fmin * 2.0**(i / bins_per_octave)
-        if freq * (1 + 1.0 / Q) > sr / 2.0:
+        if freq * (1 + 0.725 / Q) > sr / 2.0: # 0.725 for Hann window
```

Comment:
How difficult would it be to compute this value from the window function, rather than hard-code it?